projects/Kwikswap/index.js

```diff
@@ -0,0 +1,59 @@
+const { getChainTvl } = require("../helper/getUniSubgraphTvl");
+const { calculateUsdUniTvl } = require("../helper/getUsdUniTvl");
+const { staking } = require("../helper/staking.js");
+
+const v1graph = getChainTvl(
+  {
+    ethereum:
+      "https://api.thegraph.com/subgraphs/name/kwikswap/kwikswap-subgraph",
+    bsc: "https://api.thegraph.com/subgraphs/name/kwikswap/kwikswap-bsc-subgraph",
+    polygon: "https://api.thegraph.com/subgraphs/name/kwikswap/matic-exchange",
+  },
+  "kwikswapFactories",
+  "totalLiquidityUSD"
+);
+
+const KWIK_TOKEN_ADDRESSES = {
+  ethereum: "0x286c0936c7eaf6651099ab5dab9ee5a6cb5d229d",
+  shiden: "0xd67de0e0a0fd7b15dc8348bb9be742f3c5850454",
+};
+
+const STAKING_CONTRACTS = {
+  ethereum: "0x57Caec63E87e1496E946181e3Fc59086e589D4c0",
+  shiden: "0x212CB413c48221cA6fE2100578a9ABED26840380",
+};
+
+module.exports = {
+  misrepresentedTokens: true,
+  ethereum: {
+    tvl: v1graph("ethereum"),
+    staking: staking(
+      STAKING_CONTRACTS["ethereum"],
+      KWIK_TOKEN_ADDRESSES["ethereum"]
+    ),
+  },
+  shiden: {
+    tvl: calculateUsdUniTvl(

```

Comment:
missing staking on shiden?

Reply:
Got this error!
(node:6660) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'push' of undefined

![image](https://user-images.githubusercontent.com/73940078/142995411-8d604b18-a317-4202-8792-55bf3857ea82.png)